browser-test/src/landing_page.test.ts

```diff
@@ -1,22 +1,13 @@
 import {
-  AuthStrategy,
   createTestContext,
   enableFeatureFlag,
   disableFeatureFlag,
   validateAccessibility,
   validateScreenshot,
   TestContext,
 } from './support'
-import {TEST_USER_AUTH_STRATEGY} from './support/config'
 
 function sharedTests(ctx: TestContext, screenshotName: string) {
```

Comment:
The landing page (which is language selection, as of this PR) no longer has login options directly on the page; they are in the header.